SystemInfo/Collector/EzcHardwareSystemInfoCollector.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * File containing the EzcHardwareSystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use Doctrine\DBAL\Connection;
+use ezcSystemInfo;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+class EzcHardwareSystemInfoCollector implements SystemInfoCollector
```

Comment:
Tiny PhpDoc for the class. What about "Collects hardware system information using zetacomponents/sysinfo." ?